pkg/operator/ceph/config/config.go

```diff
@@ -66,6 +72,13 @@ var (
 	// VarLogCephDir defines Ceph logging directory. It is made overwriteable only for unit tests where it
 	// may be needed to send data intended for /var/log/ceph to a temporary test dir.
 	VarLogCephDir = "/var/log/ceph"
+
+	// chownUserGroup represents ceph:ceph
+	chownUserGroup = CephUser + ":" + CephGroup
+
+	// ContainerPostStartCmd is the command we run before starting any Ceph daemon
+	// It makes sure Ceph directories are owned by 'ceph'
+	ContainerPostStartCmd = []string{"chown", "--recursive", "--verbose", chownUserGroup, VarLogCephDir, VarLibCephDir}
```

Comment:
We've had issues with recursive chown on large mounts, particularly with cephfs. Can we expect the chown to run quickly so the daemon can start immediately? If the chown is only on config directories we should be fine though. What about the mons that may have several GB of metadata on disk?

Reply:
For an upgrade, daemons that are already running on directories owned by root will keep running with this user. We don't `chown` data directory, only on mon `mkfs`, so it's not an issue.